2-library/hexagonal-lib/src/main/java/com/_6core/lib/java/hexagonallib/v1/DomainService.java

```diff
@@ -0,0 +1,17 @@
+package com._6core.lib.java.hexagonallib.v1;
```

Comment:
We dont need versioning here. 
These annotations are going to be created once and forever.
Remember that package name is `groupId + artifactId`
```suggestion
package com._6core.lib.java.hexagonallib;
```